### DISCUSSION TITLE
How do I get helicopters

### DISCUSSION THREAD

**Discussion by Flameslinger02** [Sparker95/Project_0.Stratis#825] [Q&A] [answered]
I wouldn't have written this if I didn't look everywhere so if I missed something completely obvious please enlighten me : ). Do helicopters just come from reinforcements at airfields because my friends and I have gotten a save to 19 percent complete on the alpha branch and 3 on the dev but we haven't seen one heli. We have used RHS usaf and USMC so maybe that's the problem?

**Comment by Sparker95** [comment, marked as answer]
Hi, I think helicopters appear at roughly 30% aggression value. At some point they appear at airfields and enemy will sometimes use helicopters for counterattacks.

**Reply by Sparker95** [reply]
Also feel free to join our discord, link is at repository main page.

**Reply by Flameslinger02** [reply]
ok thank you so much! We did Malden so we could get to 30% faster and it worked perfectly
